test/shows.jl

```diff
@@ -4,27 +4,27 @@
 
     # compact mode
     iostr = sprint(show, T)
-    @test iostr == "Select((:a, :b, :c))"
+    @test iostr == "Select(TableTransforms.NameSpec([:a, :b, :c]))"
```

Comment:
We still need IO methods for these specs so that these tests remain unchanged.

Reply:
@juliohm takes a look at the new prints.